README.md

```diff
@@ -30,15 +39,9 @@ npm install -g grunt-cli
 npm install
 ```
 
-- Perform an initial build:
```

Comment:
That looks right. The default grunt task is also `serve` so you don't need to mention it explicitly.